http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandlerSpec.scala

```diff
@@ -191,7 +179,6 @@ class AzurePubsubHandlerSpec
         } yield {
           getRuntime.asyncRuntimeFields.flatMap(_.hostIp).isDefined shouldBe true
           getRuntime.status shouldBe RuntimeStatus.Running
-          getRuntime.auditInfo.dateAccessed.isAfter(startTime.plusMillis(mockLatencyMillis)) shouldBe true
```

Comment:
@mlilynolting, this line

Reply:
iirc this was verifying that the creation time was being recorded as when create finished, instead of when the create request was received. Is the check causing problems?